test/services/network/tracing/networktrace/actions/test_equipment_tree_builder.py

```diff
@@ -162,8 +174,15 @@ def _verify_tree_asset(
     else:
         assert tree_node.parent is None
 
-    children_nodes = list(c.identified_object for c in tree_node.children)
-    assert children_nodes == expected_children
+    children_nodes = [c.identified_object for c in tree_node.children]
+    try:
+        for child in expected_children:
+            assert child in children_nodes
+        for child in children_nodes:
+            assert child in expected_children
+    except AssertionError as e:
+        e.args = (expected_children, children_nodes)
```

Comment:
this catches the exception of any item in a not being in b and vice versa, and represents it as a != b with a comparison of all elements, i figured it was slightly more useful to see the whole difference rather then just what item we're checking not being in the other list.

Reply:
i trust you